src/value.jl

```diff
@@ -143,6 +143,11 @@ function Base.isless(x::CatValue, y::CatValue)
     end
 end
 
+Base.isless(x::CategoricalString, y::AbstractString) = order(x) < order(x.pool[get(x.pool, y)])
+Base.isless(y::AbstractString, x::CategoricalString) = !(isless(x, y) || isequal(x, y))
```

Comment:
It would probably be more efficient to define this method in terms of `>` on the order (like the one above). Same for other methods.

Reply:
OK - I forgot that for `order` we can simply use `<` and `>`.